### PR TITLE
literatureRequest: form

### PR DESCRIPTION
**Pull request by jrcastro2**
* Updates fields
* closes https://github.com/CERNDocumentServer/cds-ils/issues/355
image

### CODE REVIEW

**Review by KonstantinaStoikou — APPROVED** [review]

File: ui/src/overridden/frontsite/DocumentRequest/DocumentRequestForm.js [hunk 1]
```diff
@@ -1,11 +1,76 @@
...
+const customRequestType = {
+  placeholder: 'Select type...',
+  customLabel: (
+    <div className="display-flex required field">
+      <label className="custom-label">Request type</label>
+      <InfoPopup
+        message={
+          <>
+            <p>
+              <strong>Loan:</strong> you want to borrow or suggest a document.
```
Comment (ntarocco) [inline ui/src/overridden/frontsite/DocumentRequest/DocumentRequestForm.js:37]:
```suggestion
              <strong>Loan:</strong> you want to borrow or suggest a new literature.
```

File: ui/src/overridden/frontsite/DocumentRequest/DocumentRequestForm.js [hunk 2]
```diff
@@ -1,11 +1,76 @@
...
+              The library will provide it to you from our collections or from
+              another library. (No payment required)
+            </p>
+            <p>
+              <strong>Purchase:</strong> you want to purchase this document for
```
Comment (ntarocco) [inline ui/src/overridden/frontsite/DocumentRequest/DocumentRequestForm.js:42]:
```suggestion
              <strong>Purchase:</strong> you want to purchase this literature for
```

File: ui/src/overridden/frontsite/DocumentRequest/DocumentRequestFormHeader.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { FrontSiteRoutes } from '@inveniosoftware/react-invenio-app-ils';
+import { Link } from 'react-router-dom';
+
+export const DocumentRequestFormHeader = () => {
+  return (
+    <>
+      <Header as="h1">Request new literature</Header>
+      <p>
+        Fill in the form below to request to loan, purchase or suggest a
+        document (book, article, standard, etc.) to the Library.
```
Comment (ntarocco) [inline ui/src/overridden/frontsite/DocumentRequest/DocumentRequestFormHeader.js:12]:
```suggestion
        new literature (book, article, standard, etc.) to the Library.
```

**Review by ntarocco — APPROVED** [review]

**Review by kprzerwa — COMMENTED** [review]
check my comment in ILS please

**Pull request merged**